model-providers/openai/openai-vanilla/runtime/src/main/java/io/quarkiverse/langchain4j/openai/runtime/OpenAiRecorder.java

```diff
@@ -59,7 +59,8 @@ public Supplier<ChatLanguageModel> chatModel(LangChain4jOpenAiConfig runtimeConf
                     .topP(chatModelConfig.topP())
                     .presencePenalty(chatModelConfig.presencePenalty())
                     .frequencyPenalty(chatModelConfig.frequencyPenalty())
-                    .responseFormat(chatModelConfig.responseFormat().orElse(null));
+                    .responseFormat(chatModelConfig.responseFormat().orElse(null))
+                    .stop(chatModelConfig.stop().orElse(null));
```

Comment:
This should also be done for `streamingChatModel()` too, right?

Reply:
Indeed ! Let me add it (And it's even in streaming mode where this stop sequence will make the most sense )